block/manager.go

```diff
@@ -228,6 +230,22 @@ func (m *Manager) SyncLoop(ctx context.Context) {
 			if err != nil {
 				m.logger.Info("failed to sync next block", "error", err)
 			}
+		case fraudProof := <-m.FraudProofCh:
+			m.logger.Debug("fraud proof received", "Block Height", "dummy block height") // TODO: Insert real block height
+			_ = fraudProof
+			// TODO(light-client): Set up a new cosmos-sdk app
+			// How to get expected appHash here?
+
+			// success, err := m.executor.VerifyFraudProof(fraudProof, nil)
```

Comment:
Isn't this code essential for this PR?

Reply:
This isn't possible currently because we don't have a fraud proof window implemented. Made a tracking issue for it [here](https://github.com/celestiaorg/rollmint/issues/549).